src/Components/SearchBar.js

```diff
@@ -44,10 +48,13 @@ const SearchBar = ({
         )}
         value={userInputSearchBar}
         onChange={e => dispatch({ type: 'HANDLE_SEARCHBAR_INPUT', userInputSearchBar: e.target.value })}
-        onSelect={value3 => axios.get(`${process.env.REACT_APP_API_URL}/pois/filter/${value3}`)
-          .then(res => dispatch({ type: 'HANDLE_SELECT', userInputSearchBar: value3, filteredPoiByKeyword: res.data }))}
+        onSelect={value3 => (filter(poiKeywordsDisplay, value3).length
+          ? axios.get(`${process.env.REACT_APP_API_URL}/pois/filterKeyword1/${value3}`)
```

Comment:
It would be better to put the condition in the string directly to avoid code repetition